packages/open-next/src/wrappers/aws-lambda-streaming.ts

```diff
@@ -23,7 +23,12 @@ function formatWarmerResponse(event: WarmerEvent) {
 
 const handler: WrapperHandler = async (handler, converter) =>
   awslambda.streamifyResponse(
-    async (event: AwsLambdaEvent, responseStream): Promise<AwsLambdaReturn> => {
+    async (
+      event: AwsLambdaEvent,
+      responseStream,
+      context,
+    ): Promise<AwsLambdaReturn> => {
+      context.callbackWaitsForEmptyEventLoop = false;
```

Comment:
Didn't Dax add this to the `sst` layer at one point, which broken `open-next`?

Reply:
Did he ? He did it in the remix server in ion, and at the time i tried it in OpenNext as well ( to fix the 30s hanging issue ), but it ended up causing issue with ISR because of the pending promise ( at the time i didn't know that ).

Reply:
I may have misremembered, but iirc he added that to the base SSR construct in `sst`. Things may be changed now, but it was many months ago.

Reply:
You're probably right actually, i thought you were talking about the remix stuff. 
Anyway it works now